pkg/resources/pool_stub.go

```diff
@@ -68,6 +68,11 @@ func (rp *resourcePool) GetDeviceSpecs(deviceIDs []string) []*pluginapi.DeviceSp
 	for _, id := range deviceIDs {
 		if dev, ok := rp.devicePool[id]; ok {
 			newSpecs := dev.GetDeviceSpecs()
+			rdmaSpec := dev.GetRdmaSpec()
+			if rp.config.IsRdma && rdmaSpec.IsRdma() {
```

Comment:
what will happen if a resource is marked as isRdma in the config file but rdmaSpec.IsRdma() is false ?
I think we should at least  log the error as the container will have no RDMA capabilities although the user requested it.

Reply:
I will add a new filter in the `deviceSelector,go` to fix it

Reply:
Also i recommend to log an error if we get to this point and (rp.config.IsRdma != rdmaSpec.IsRdma())

Reply:
ok